man/fi_provider.3.md

```diff
@@ -184,6 +184,12 @@ of the service or resource to which they correspond.
   used by the different providers.  Additional information on the cache
   is available in the `fi_mr(3)` man page.
 
+*logging*
+: The logging object references the internal logging subsystem used by
+  the different providers.  Once opened, custom logging callbacks may
+  be installed.  Can be opened only once and only the last import is
+  used if imported multiple times.
```

Comment:
We now have 2 different approaches to override logging functions:
- fi_open() + fi_import(): multiple calls to fi_import() is allowed and only the last imported object is used.
- fi_import_log() (the helper function) which would fail if called multiple times. 

My comment above was for the combination of fi_open() + fi_import(), which is valid I think.